### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#459] [Esami & Esoneri M-Z]
## Traccia

<img width="580" alt="image" src="https://github.com/user-attachments/assets/d1af5855-6f1b-47e1-b2c1-36f63a06b04e">

**Comment by rimaout** [comment]
<img width="900" alt="image" src="https://github.com/user-attachments/assets/ce3bfbf4-92f2-4912-81f2-acf21c9a963c">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/5c19189f-93ef-447a-b3e3-463e7f1389df)
